easybuild/easyconfigs/t/Trinity/Trinity-2.15.1-foss-2023a.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     Custom
+#
+# Notes::
+##
+
+name = 'Trinity'
+version = '2.15.1'
```

Comment:
@laraPPr Any reason we're not using `2.15.2`?

Reply:
No, I must have overlooked it. Bumping it